runtime/jcl/common/thread.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1998, 2020 IBM Corp. and others
+ * Copyright (c) 1998, 2022 IBM Corp. and others
```

Comment:
This file should define `Java_java_lang_Thread_yield0()`, not `Java_java_lang_Thread_yield()` to be consistent with the export list and the declaration in `jclprots.h`.